app/components/assets/form.tsx

```diff
@@ -163,7 +180,51 @@ export const AssetForm = ({
         </FormRow>
       </div>
 
-      <div className="text-right">
+      <div className="border-b pb-6">
+        <div className="mb-6 border-b pb-5">
+          <h2 className="mb-1 text-[18px] font-semibold">Custom Fields</h2>
+          <Link
+            to="/settings/custom-fields"
+            className="font-medium text-primary-600"
+          >
+            Manage custom fields
+          </Link>
+        </div>
+        {fields.length > 0 ? (
+          fields.map((field) => (
+            <FormRow
+              key={field.id}
+              rowLabel={field.name}
+              subHeading={field.helpText && <p>{field.helpText}</p>}
+              className="border-b-0"
+            >
+              <Input
+                hideLabel
+                type={field.type}
+                label={field.name}
+                name={field.name}
+                className="w-full"
+              />
+            </FormRow>
+          ))
+        ) : (
+          <div>
```

Comment:
Lets move this to its own component so we can keep the file cleaner. Its already quite big.

Reply:
sure